README.md

```diff
@@ -13,39 +13,119 @@ Artifacts in support of ODH Edge use cases that show integration with Red Hat Ad
-   ```
-   $  oc apply -k acm/registration/
-   ```
-   * Core - Cluster host the ODH Core components that will be used in the MLOps Engineer workflow to train, build and push the model.  This cluster is not required to be co-located with the ACM Hub but we group them together to simplify the use case
-   * Near Edge - Cluster(s) that will host the running model at the edge.  This is the target environment after a new model is available for use
-1. Deploy Open Data Hub to the Core cluster and register any configurations to support pushing models to the edge cluster
-   * Open Data Hub - v2.x of the operator will be installed as part of the dependencies for the core cluster.  Manual installation of the [DataScienceCluster](https://github.com/opendatahub-io/opendatahub-operator#example-datasciencecluster) object will be required to deploy the Open Data Hub core stack to support the data science workflow
-   * GitOps repos
-   * Image repos
-1. Manage the edge cluster environments to support deployment of models and support for monitoring
-   * Configure ACM Observability
-   * Deploy the Model container
-
-### MLOps Engineer workflows
-
-1. Develop the model in an ODH Jupyter notebook
-1. Build the model from the notebook using Data Science Pipelines
-1. Push the model to the image registry accessible by the near edge cluster(s)
-1. Update the GitOps config for the near edge cluster
-
-### Pipelines setup
-
-See [pipelines/README.md](pipelines/README.md)
+Working and deploying your own models might require bigger changes
+to the definition and configuration of the pipelines and ACM setup below,
+so you might want to start with the pre-built models first.
+
+### Building container image with model and serving runtime
+
+In [pipelines/README.md](pipelines/README.md) we show how to take the trained models,
+store them in a S3 bucket,
+build container image(s) with the model and serving runtime using OpenShift Pipelines,
+push the container image(s) to an image registry accessible by the near edge cluster(s),
+and update a clone of this repository with a pull request,
+configuring `acm/odh-edge/apps/*/kustomization.yaml` with location and digest (SHA-256) of the built images.
+
+You can skip this step if you do not wish to rebuild the container images.
+If you use the default configuration as shown in this git repository,
+you will use already built container images from the https://quay.io/organization/rhoai-edge repositories.
+
+### Serving to near edge remote clusters
+
+We assume that you have an admin access to an OpenShift cluster where you can install and configure
+the Red Hat Advanced Cluster Management operator and which will serve as the central ACM hub,
+and other OpenShift cluster(s) with admin privileges that you will import to the ACM hub cluster
+and that will play the role of near edge location(s), managed from the central ACM hub.
```

Comment:
```suggestion
We assume that you have admin access to an OpenShift cluster where you can install and configure
the Red Hat Advanced Cluster Management operator and which will serve as the central ACM hub.
We also assume you have admin access to other OpenShift cluster(s) that you will import to the ACM hub cluster, these will play the role of near edge location(s) managed from the central ACM hub.
```

Reply:
Applied in 49aadf7f7022a32884be3d91d13200f348ad82ab.